c2corg_ui/__init__.py

```diff
@@ -68,3 +73,19 @@ def _add_static_view(config, name, path):
     )
     config.add_cache_buster(path, version_cache_buster)
     CACHE_PATH.append(name)
+
+
+class NotFound():
+
+    notFound = 'c2corg_ui:templates/404.html'
+
+    def __init__(self, request):
+        self.settings = request.registry.settings
+
+    @view_config(context=HTTPInternalServerError, renderer=notFound)
+    @view_config(context=HTTPBadRequest, renderer=notFound)
+    @notfound_view_config(renderer=notFound)
+    def notfound(self):
+        return {
```

Comment:
This view seems to always return a status code 200 even though the caught error is 400/404/500 => the tests fail because the expected code is not returned.

Should we change the tests or forward the status code retrieved in some response object?